src/riak_cs_gc_key_list.erl

```diff
@@ -180,6 +182,46 @@ gc_index_query(RcPid, StartKey, EndKey, BatchSize, Continuation, UsePaginatedInd
 
     {QueryResult, {StartKey, EndKey}}.
 
+-spec find_oldest_entries(BagId::binary()|master) ->
+                                 {ok, [{string(), [char()]}]} |
+                                 {error, term()}.
+find_oldest_entries(BagId) ->
+    %% walk around
+    {ok, RcPid} = riak_cs_riak_client:start_link([]),
+    ok = riak_cs_riak_client:set_manifest_bag(RcPid, BagId),
+    Start = riak_cs_gc:epoch_start(),
+    End = riak_cs_gc:default_batch_end(riak_cs_gc:timestamp(),
+                                       riak_cs_gc:leeway_seconds()),
```

Comment:
Do we need to consider leeway period here?
One reason that came into my head is because the GC batch command has `end` and `leeway` options and users can garbage-collect younger objects than leeway.